pkg/broker/user_info.go

```diff
@@ -0,0 +1,62 @@
+package broker
+
+import (
+	"encoding/json"
+	"fmt"
+
+	osb "github.com/pmorie/go-open-service-broker-client/v2"
+)
+
+// ParseKubernetesIdentity - creates a kubernetes identity from the
+// orginating identity
+func ParseKubernetesIdentity(o osb.OriginatingIdentity) (*K8SIdentity, error) {
+	if o.Platform != "kubernetes" {
```

Comment:
I made a PR to add constants for these to the client library: https://github.com/pmorie/go-open-service-broker-client/pull/111

Reply:
I actually have a quick question about where the types are. To me, it seems that we should either have a shared place for the types or the types should be in the lib. It is mostly about where I would expect to find the types if I was looking. Just my two cents, and was wondering if that was a thought and not what we started with, or if you actually thought that was the wrong way to think about it?

Reply:
I think the types should be here, because these are for consumption on the broker side. On the client side (in service-catalog), we serialize a k8s `AuthInfo` directly to get the value we send on the client request. Adding a k8s identity in go-open-service-broker-client doesn't really help us there.

We can always add the type here and move it to the client later if there's a demand for it.

Reply:
@pmorie re: pmorie/go-open-service-broker-client/pull/111 should I update the dependency in this PR or should another PR be opened to update vendor?

Reply:
Added PR: #9 to bump the client to get the new constants.